src/%IPM/CLI.cls

```diff
@@ -50,7 +50,6 @@ ClassMethod %Help(ByRef pCommandInfo) [ Final ]
 		// List commands
 		If 'tMarkdown {
 			Write !, $$$FormattedLine($$$Underlined, "Available commands:")
-			Write !, "NOTE: [] around a parameter indicates it is optional"
```

Comment:
@Beiming-Zhang For this class in particular (and perhaps some other classes as well), for this merge, could you revert to using the existing indentation so it is easier to see the diffs? Could fix just the indentation across the repo in a separate PR